api/v1alpha1/limitador_types.go

```diff
@@ -317,18 +321,3 @@ func (s *LimitadorStatus) Equals(other *LimitadorStatus, logger logr.Logger) boo
 func init() {
 	SchemeBuilder.Register(&Limitador{}, &LimitadorList{})
 }
-
-type PodDisruptionBudgetType struct {
```

Comment:
moving this type to have `func init() {` as the last function in the file. Just personal taste.